docs/development.md

```diff
@@ -321,3 +321,30 @@ Inspired from: [github.com/kubernetes/community/contributors/devel/api-conventio
   - Add an example in docs/example with its explanation README, for e.g. [health](https://github.com/kedgeproject/kedge/tree/master/docs/examples/health).
   - Add an e2e test on above example, for e.g. see test code for [health](https://github.com/kedgeproject/kedge/blob/cfee15ffde02c611d08420699a43869706be2d53/tests/e2e/e2e_test.go#L272).
   - Add this feature information to file-reference, for e.g. see [health section](https://github.com/kedgeproject/kedge/blob/master/docs/file-reference.md#health).
+
+### golang dependency import conventions
+
+imports MUST be arranged in three sections, separated by empty line.
+
+```go
+stdlib
+kedge
+thirdparty
```

Comment:
We could also add a named import section here like done in https://github.com/kedgeproject/kedge/blob/master/pkg/spec/resources.go#L19-L38, this makes the imports look a lot cleaner.

Reply:
I think it is for knowing what imports are coming from where, also even if the imports are named or not the sorting happens the same way.

And same convention k8s also follows.

Reply:
You should use named imports only when there is a collision between multiple package names. Otherwise, it just adds  confusion